tests/lib/ocpp/v201/test_charge_point.cpp

```diff
@@ -261,6 +288,68 @@ class ChargePointFixture : public DatabaseTestingUtils {
  * is provided.
  */
 
+TEST_F(ChargePointFixture, CreateChargePoint) {
```

Comment:
Please move these tests above the comment above, because this comment addresses the `K01.FR.02` states

Reply:
Addressed.